tests/test_graph.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for graph module."""
+
+import networkx as nx
+
+import causalprog
+
+
+def test_label2() -> None:
+    """Test nodes."""
+    family = causalprog.graph.node.DistributionFamily()
+    node = causalprog.graph.RootDistributionNode(family, "N0")
+    node2 = causalprog.graph.RootDistributionNode(family, "N1")
+    node3 = causalprog.graph.RootDistributionNode(family, "Y")
+    node4 = causalprog.graph.DistributionNode(family, "N4")
+    node_copy = node
+
+    assert node.label == node_copy.label
+    assert node.label != node2.label
+    assert node.label != node3.label
+    assert node.label != node4.label
+
+    assert isinstance(node, causalprog.graph.node.Node)
+    assert isinstance(node2, causalprog.graph.node.Node)
+    assert isinstance(node3, causalprog.graph.node.Node)
+    assert isinstance(node4, causalprog.graph.node.Node)
+
+
+def test_simple_graph() -> None:
+    """Test a simple graph."""
+    family = causalprog.graph.node.DistributionFamily()
+    n_x = causalprog.graph.RootDistributionNode(family, "N_X")
+    n_m = causalprog.graph.RootDistributionNode(family, "N_M")
+    u_y = causalprog.graph.RootDistributionNode(family, "U_Y")
+    x = causalprog.graph.DistributionNode(family, "X")
+    m = causalprog.graph.DistributionNode(family, "M")
+    y = causalprog.graph.DistributionNode(family, "Y", is_outcome=True)
```

Comment:
We probably want to bundle this guy up into a fixture at some point... I imagine we will be using `simple_graph` in a lot of tests going forward!